src/js/views/admin/program/workflows/workflows_views.js

```diff
@@ -82,14 +87,58 @@ const LayoutView = View.extend({
       regionClass: PreloadRegion,
       replaceElement: true,
     },
+    add: '[data-add-region]',
   },
   template: LayoutTemplate,
   triggers: {
     'click .js-add': 'click:add',
   },
 });
 
+const AddActionDroplist = Droplist.extend({
+  initialize({ model }) {
+    this.collection = new Backbone.Collection([
+      {
+        onSelect(program) {
+          Radio.trigger('event-router', 'program:action:new', program.id); // probably not gonna work?
+        },
+        id: 'new-action',
+        isFas: false,
+        icon: 'file-alt',
+        name: intl.admin.program.workflows.newAction,
+      },
+      {
+        onSelect(program) {
+        },
+        id: 'new-flow',
+        isFas: true,
+        icon: 'folder',
+        name: intl.admin.program.workflows.newFlow,
+      },
+    ]);
+  },
+  popWidth: 248,
+  picklistOptions() {
+    return {
+      attr: 'name',
+      className: 'picklist workflows__picklist',
+      headingText: 'Add Workflow',
+      lists: [{
```

Comment:
so because there's one list this can be:
```
attr: 'name',
className: 'picklist workflows__picklist',
headingText: 'Add Workflow'
itemTemplate: hbs``
```
And droplist.collection is automatically handled as the droplist's primary list collection, so not needed to be attached.